From 459727cb2018b4e0a15f0827369197f18722497e Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 6 Sep 2016 14:50:07 -0700
Subject: [PATCH] DOCKER-926 node-docker-registry-client is failing 'make
 check' Reviewed by: Todd Whiteman <todd.whiteman@joyent.com> Approved by:
 Todd Whiteman <todd.whiteman@joyent.com>

---
 lib/registry-client-v1.js |  4 ++--
 lib/registry-client-v2.js | 12 ++++++------
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/lib/registry-client-v1.js b/lib/registry-client-v1.js
index 08c460e..e675149 100644
--- a/lib/registry-client-v1.js
+++ b/lib/registry-client-v1.js
@@ -125,9 +125,9 @@ function login(opts, cb) {
     assert.func(cb, 'cb');
 
     var index = opts.index || opts.indexName;
-    if (typeof(index) === 'string') {
+    if (typeof (index) === 'string') {
         try {
-            var index = common.parseIndex(index);
+            index = common.parseIndex(index);
         } catch (indexNameErr) {
             cb(indexNameErr);
             return;
diff --git a/lib/registry-client-v2.js b/lib/registry-client-v2.js
index aad58e0..49f68fb 100644
--- a/lib/registry-client-v2.js
+++ b/lib/registry-client-v2.js
@@ -612,9 +612,9 @@ function ping(opts, cb) {
     // assert.optional object or bool(opts.proxy, 'opts.proxy');
 
     var index = opts.index || opts.indexName;
-    if (typeof(index) === 'string') {
+    if (typeof (index) === 'string') {
         try {
-            var index = common.parseIndex(index);
+            index = common.parseIndex(index);
         } catch (indexNameErr) {
             cb(indexNameErr);
             return;
@@ -754,9 +754,9 @@ function login(opts, cb) {
     assert.func(cb, 'cb');
 
     var index = opts.index || opts.indexName;
-    if (typeof(index) === 'string') {
+    if (typeof (index) === 'string') {
         try {
-            var index = common.parseIndex(index);
+            index = common.parseIndex(index);
         } catch (indexNameErr) {
             cb(indexNameErr);
             return;
@@ -944,8 +944,8 @@ function RegistryClientV2(opts) {
     }
     if (opts.scheme) {
         this.repo.index.scheme = opts.scheme;
-    } else if (!this.repo.index.scheme
-        && common.isLocalhost(this.repo.index.name))
+    } else if (!this.repo.index.scheme &&
+        common.isLocalhost(this.repo.index.name))
     {
         // Per docker.git:registry/config.go#NewServiceConfig we special
         // case localhost to allow HTTP. Note that this lib doesn't do
-- 
2.21.0

